eng/scripts/Copy-ApiVersion.ps1

```diff
@@ -49,7 +49,15 @@ param (
     [string] $NewVersion,
     
     [Parameter(Mandatory = $true)]
-    [ValidateScript({ Get-Version $_ })]
+    [ValidateScript({ 
```

Comment:
Why did you change this? It reuses the `Get-Version` function defined in the script scope above so we don't duplicate.